org.civicrm.reqangular/src/common/services/crm-ang.service.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env amd */
+
+(function (CRM) {
+  define([
+    'common/lodash',
+    'common/modules/services'
+  ], function (_, services) {
+    'use strict';
+
+    services.factory('crmAngService', function () {
+      return {
+        loadForm: loadForm
+      };
+    });
+
+    /**
+     * Opens a CRM form with a provided url and options
+     * This is simply a wrapper to CRM.loadForm
+     * Reference: https://docs.civicrm.org/dev/en/latest/framework/ajax
+     *
+     * @param  {String} url
+     * @param  {Object} options
+     * @return {jQuery}
```

Comment:
looks like it's returning the jQuery library

Reply:
Done.